### PR TITLE
Update config for mainnet

### PR DESCRIPTION
**Pull request by danieledamiani**

### CODE REVIEW

File: packages/tempus-client_v2/src/config/config.ts [hunk 1]
```diff
@@ -1,19 +1,59 @@
...
+// const GOERLI_ALCHEMY_KEY = process.env.REACT_APP_GOERLI_ALCHEMY_KEY || '';
+// const GOERLI_ALCHEMY_CONFIG = {
+//   networkUrl: 'https://eth-goerli.alchemyapi.io/v2/RUE8LXWaELu4_nyMgfK99eX3DWoxLtDw',
+//   networkName: 'goerli',
+//   alchemyKey: GOERLI_ALCHEMY_KEY,
+// } as Config;
+
+const MAINNET_ALCHEMY_KEY = process.env.REACT_APP_MAINNET_ALCHEMY_KEY || '';
+const MAINNET_ALCHEMY_CONFIG = {
+  networkUrl: 'https://eth-mainnet.alchemyapi.io/v2/CikBd5PRY8HTC1rBQw1dBq5HlOfXwe0O',
```
Comment (lazarmitic) [inline packages/tempus-client_v2/src/config/config.ts:12]: we should probably hide alchemy key from this URL as well.

File: packages/tempus-client_v2/src/services/NotificationService.ts [hunk 2]
```diff
@@ -117,10 +116,11 @@ class NotificationService {
...
 
 export const generateEtherscanLink = (tx: string) => {
-  const config = getConfig();
-
+  // const config = getConfig();
   // TODO - No need to specify network name for mainnet
-  return `https://${config.networkName}.etherscan.io/tx/${tx}`;
+  // return `https://${config.networkName}.etherscan.io/tx/${tx}`;
+
+  return `https://.etherscan.io/tx/${tx}`;
```
Comment (lazarmitic) [inline packages/tempus-client_v2/src/services/NotificationService.ts:123]: there is an extra . before etherscan.

**Review by lazarmitic — CHANGES_REQUESTED** [review]

File: packages/tempus-client_v2/src/config/config.ts (hunk 1 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/config/config.ts:12]: right

File: packages/tempus-client_v2/src/services/NotificationService.ts (hunk 2 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/services/NotificationService.ts:123]: good spot

**Comment by danieledamiani** [comment]
updated

**Review by lazarmitic — APPROVED** [review]

**Pull request merged**